ord_schema/proto/reaction.proto

```diff
@@ -196,6 +196,7 @@ message CrudeComponent {
  * be provided about how it was prepared and from where it was purchased.
  */
 message Compound {
+  reserved 13;
```

Comment:
Do you want to drop the `reserved` keys and renumber everything? This should be fine if you're copying reaction.proto.

Reply:
I'll defer to you and @antonkast-google on protobuf best practices here. I kind of like the reserved keys so there's no risk of reading an old/outdated message, not that the risk is terribly high to begin with. The change that makes me the most nervous are the ones to enums, since they're just passed around as integers in the compiled Python code and it would be very easy to incorrectly migrate those (or forget to).

Reply:
Has anyone forked this repo? Do we know of anyone holding on to protos somewhere?

If not, I see no harm either way.

Reply:
I think that since there's no way to make a clean transition from the old to the new (correct me if I'm wrong there; renaming fields breaks `pbtxt` and reusing existing tag numbers breaks `pb`), we should drop the reserved fields and renumber things now so that when things break, they are more likely to break noisily.

Said another way: Using `reserved` and assigning new tag numbers does keep the `pb` files working, but they are not the source of truth---the `pbtxt` files are.

People holding on to old protos can use an earlier version of ord-schema to read them. We'll include details about the break when we release a version with the new changes (and possibly increment the major version to 1?).

Reply:
SGTM